dashbuilder-shared/dashbuilder-dataset-api/src/main/java/org/dashbuilder/dataprovider/BeanProviderType.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dashbuilder.dataprovider;
+
+import org.dashbuilder.dataset.def.SQLDataSetDef;
+import org.dashbuilder.dataset.json.DataSetDefJSONMarshallerExt;
+import org.dashbuilder.dataset.json.SQLDefJSONMarshaller;
+
+/**
+ * For accessing data sets generated directly from a bean class implementing the DataSetGenerator interface
+ */
+public class BeanProviderType implements DataSetProviderType<SQLDataSetDef> {
```

Comment:
Yeah!

good point... it was a temporal commit. The PR is not completed, still
working on it...

Thanks anyway.

On Thu, May 12, 2016 at 6:29 PM, Jan Hrcek <notifications@github.com> wrote:

> In
> dashbuilder-shared/dashbuilder-dataset-api/src/main/java/org/dashbuilder/dataprovider/BeanProviderType.java
> <https://github.com/dashbuilder/dashbuilder/pull/180#discussion_r63051922>
> :
>
> > + * Unless required by applicable law or agreed to in writing, software
> > + * distributed under the License is distributed on an "AS IS" BASIS,
> > + * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > + * See the License for the specific language governing permissions and
> > + * limitations under the License.
> > + */
> > +package org.dashbuilder.dataprovider;
> > +
> > +import org.dashbuilder.dataset.def.SQLDataSetDef;
> > +import org.dashbuilder.dataset.json.DataSetDefJSONMarshallerExt;
> > +import org.dashbuilder.dataset.json.SQLDefJSONMarshaller;
> > +
> > +/**
> > + * For accessing data sets generated directly from a bean class implementing the DataSetGenerator interface
> > + */
> > +public class BeanProviderType implements DataSetProviderType<SQLDataSetDef> {
>
> BeanProviderType implementes DataSetProviderType looks suspicious. Also
> getName returns "SQL"..
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/dashbuilder/dashbuilder/pull/180/files/e4e2512d6dc698552efbe395908367dbbe2bf775#r63051922>
>